src/Umbraco.AuthorizedServices/Services/Implement/AuthorizedServiceCaller.cs

```diff
@@ -100,6 +104,20 @@ public async Task<string> SendRequestRawAsync<TRequest>(string serviceAlias, str
                 apiKey,
                 requestContent);
         }
+        else if (serviceDetail.AuthenticationMethod == AuthenticationMethod.OAuth1)
+        {
+            OAuth1aToken? token = GetOAuth1aToken(serviceAlias);
+            if (token is null)
+            {
+                // No token exists, query the service for a request_token.
```

Comment:
The main reason behind this is to avoid making calls for a `request_token` to the service each time the service details are retrieved in case the authorization process completed successfully already and an `access_token` and `token_secret` exist.